test/System.IdentityModel.Tokens.Jwt.Tests/JwtSecurityTokenHandlerTests.WithContextSwitches.cs

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+﻿using System.Security.Claims;
+using Microsoft.IdentityModel.JsonWebTokens;
+using Microsoft.IdentityModel.TestUtils;
+using Microsoft.IdentityModel.Tokens;
+using Xunit;
+
+namespace System.IdentityModel.Tokens.Jwt.Tests
+{
+    [Collection(nameof(JwtSecurityTokenHandlerNonParallelRunTests))]
+    public class JwtSecurityTokenHandlerTestsWithContextSwitches
+    {
+        [Fact]
+        public void JwtSecurityTokenHandler_CreateToken_AddShortFormMappingForRsaOAEPEnabled()
```

Comment:
You can combine these tests using `InlineData` - (bool enableSwith, string expectedAlgorithm)

https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/909669f4053f485ad94223fa51e2b8be2f61a76f/test/Microsoft.IdentityModel.Tokens.Tests/CryptoExtensibilityTests.cs#L151

Reply:
You're right, forgot about inlinedata. Thanks Peter!